src/components/CourseList/CourseItem.js

```diff
@@ -56,13 +56,13 @@ CourseItem.propTypes = {
   // mapStateToProps
   t: PropTypes.func.isRequired,
   language: PropTypes.string.isRequired,
-  showPlaylists: PropTypes.bool.isRequired,
+  hideLessonCount: PropTypes.bool.isRequired,
 };
 
 const mapStateToProps = (state) => ({
   t: getTranslator(state),
   language: state.language,
-  showPlaylists: state.showPlaylists,
+  hideLessonCount: state.showPlaylists && state.language === 'nb',
```

Comment:
Dette er ikke nok. Du har glemt å sjekke for om playlistlength > 0.

Forslag: I LessonFilter i `mapStateToProps` så har du linjen
```
showFiltergroups: !state.showPlaylists || !showRadiobuttons,
```
Lag en selector for denne logikken på samme måte som `getShowRadiobuttons` (og gjerne på samme sted, med mindre du har et forslag til et bedre sted). F.eks.
```
export const getShowFiltergroups = (state, courseName) =>
  !state.showPlaylists || !getShowRadiobuttons(state, courseName);
```

Så kan LessonFilter sin mapStateToProps blir:
```
const mapStateToProps = (state, {courseName}) => ({
  filterGroupKeys: Object.keys(state.filter),
  isStudentMode: state.isStudentMode,
  t: getTranslator(state),
  showRadiobuttons: getShowRadiobuttons(state, courseName),
  showFiltergroups: getShowFiltergroups(state, courseName),
});
```

Og så blir CourseItem sin:
```
const mapStateToProps = (state) => ({
  t: getTranslator(state),
  language: state.language,
  showLessonCount: getShowFiltergroups(state),
});
```